app/src/main/res/layout/item_last_movie_simple.xml

```diff
@@ -4,7 +4,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_marginBottom="100dp"
-    app:cardBackgroundColor="@color/purple_200"
+    app:cardBackgroundColor="#F4A61F"
```

Comment:
extract the color into colors.xml from values folder and replace the other places with the color reference instead of passing the hardcoded value. This way if you want to change the color you have to modify only in one place